### PR TITLE
Add install Photon OS graph

### PR DESCRIPTION
**Pull request by cgx027**
Photon OS bootstrap support

This is one of three PRs for Photon OS bootstrap:
RackHD/on-tasks #275  https://github.com/RackHD/on-tasks/pull/275
RackHD/on-taskgraph #138 https://github.com/RackHD/on-taskgraph/pull/138 
RackHD/on-http #362 https://github.com/RackHD/on-http/pull/362

@RackHD/corecommitters @iceiilin @pengz1 @WangWinson

### CODE REVIEW

**Comment by yyscamper** [comment]
:+1:

**Comment by iceiilin** [comment]
:+1:

**Comment by cgx027** [comment]
test this please

File: spec/lib/graphs/install-photon-graph-spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2015, EMC, Inc.
```
Comment (WangWinson) [inline spec/lib/graphs/install-photon-graph-spec.js]: aha ! copy & paste issue , Copyright 2016

File: spec/lib/graphs/install-photon-graph-spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    base.before(function (context) {
+        context.taskdefinition = helper.require('/lib/graphs/install-photon-graph.js');
+    });
+
+    describe('graph', function () {
+        base.examples();
+    });
+
+});
```
Comment (WangWinson) [inline spec/lib/graphs/install-photon-graph-spec.js]: new line ?

**Comment by WangWinson** [comment]
:+1:

**Comment by cgx027** [comment]
test this please

**Comment by yyscamper** [comment]
test this please

**Comment by cgx027** [comment]
test this please

**Pull request merged**